pxf/build.gradle

```diff
@@ -310,6 +315,7 @@ project('pxf-service') {
 
     project.distTar {
         from('src/main/resources/pxf-profiles.xml') { into 'conf' }
+        from('src/main/resources/pxf-profiles-default.xml') { into 'conf' }
```

Comment:
This may not work because pxf-profiles-default.xml has to be copied to pxf-profiles.xml

Reply:
For the product's perspective why is that needed ?

Reply:
So, people would have to manually copy it then?

Reply:
Not really. The purpose of pxf-profile-default.xml is to have all the default profiles. The users need to only add custom profiels to pxf-profile.xml. The PXF application loads both the xml files during startup.

Reply:
Then why did it fail for pxf-automation on HAWQ?

Reply:
Automation test failure is a different issue and that needs to be addressed separately out of this PR. This would not be an end user issue.
The automation test assumes pxf-profile.xml has all the profiles in it which is not really the case anymore. The tests need to addressed separately.

Reply:
I still do not understand why pxf-profile.xml needs to be empty but that is a product question for @kongyew

Reply:
@sansanichfb just verified in the code that we are loading both profiles.

Reply:
@lavjain pxf-profiles.xml is better to be empty to prevent users from incorrectly updating the default entires in it. This is similar to the separation we have between pxf-private.classpath from pxf-public.classpath.
@lavjain @sansanichfb  does that mean we don't need pxf-profiles.default ?